.travis.yml

```diff
@@ -3,10 +3,17 @@ language: php
 php:
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
+matrix:
+    include:
+        - php: 5.5
+          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
 before_script:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer self-update
```

Comment:
Got a link?

As far as I see it its bad, this version may not even have caret support yet, depending on how old it is. @seldaek recommends always updating composer to the latest since its not stable yet, i'll stick with him.